### ISSUE TITLE
Is TreeView supported?

### ISSUE THREAD

**Issue by mortezae** [romgrk/node-gtk#105]
Hello. I tried to adopt some PyGTK/GTKmm treeview examples for node-gtk but had no success. I get some errors but I'm not sure if it's my fault or library. Does is basically support this feature at all?

**Comment by romgrk** [comment]
Hi! Any feature should be supported. If you encounter issues, please post a minimal test case that reproduces the issue and I will look into it :)

**Comment by romgrk** [comment]
So I just bumped into this issue again, and i tried to make it work by myself. Bottom line is: it's really harder than it should be!

I've included an example of how to get it working here:
https://github.com/romgrk/node-gtk/blob/master/examples/gtk-tree-view.js
![Screenshot from 2019-04-05 17-55-24](https://user-images.githubusercontent.com/1423607/55658800-eba64c80-57cc-11e9-9f69-d98c9ebb0914.png)
(on a different subject, if you're looking for good books these ones are pretty good)

I'll close this issue as it's solved but I've opened #108 to make the API easier to use. Don't hesitate to re-open this or another issue if you have problems with this.
